mne/io/reference.py

```diff
@@ -500,6 +500,9 @@ def set_bipolar_reference(inst, anode, cathode, ch_name=None, ch_info=None,
         logger.info('Bipolar channel added as "%s".' % name)
         inst.info._update_redundant()
 
+    if getattr(inst, 'picks', None) is not None:
+        del inst.picks  # picks cannot be tracked anymore
```

Comment:
this is new for me, not sure what this does

also not [found in the Attributes section of the docstring](https://mne.tools/dev/generated/mne.Epochs.html#mne.Epochs)

Reply:
the .picks attribute when you passed picks to the Epochs constructor.